cognite_toolkit/cdf_tk/templates/_templates.py

```diff
@@ -376,7 +376,13 @@ def process_config_files(
             content = replace_variables(content, local_config)
             filename = create_file_name(filepath, number_by_resource_type)
 
-            destination = build_dir / filepath.parent.name / filename
+            try:
+                _, resource_folder = module_from_path(filepath, return_resource_folder=True)
```

Comment:
This seems like a weird overloading of functionality for module_from_path that is not quite intuitive?

Reply:
Good point, making a separate utility function.